packages/vuetify/src/composables/size.ts

```diff
@@ -0,0 +1,33 @@
+// Setup
+import { computed, Prop } from 'vue'
+
+// Types
+export interface SizeProps {
+  size: Size
+}
+
+const allowedSizes = ['x-small', 'small', 'default', 'large', 'x-large'] as const
+
+type Size = typeof allowedSizes[number] | undefined
+
+// Props
+export function sizeProps (defaults: Partial<SizeProps> = { size: 'default' }) {
```

Comment:
Are we going with `fooProps` or `makeFooProps`?

Reply:
make